cloud-claim-check-pattern/license-plugin-header-style.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
This file is unncessary. Please review and remove.

Reply:
@ohbus removed.